lib/eldap/src/eldap.erl

```diff
@@ -36,6 +36,54 @@ Type definitions that are used more than once in this module:
 - **`referrals() =`** - `[Address = string()]` The contents of `Address` is
   server dependent.
 """.
+
+-type connection_info() :: #{socket := ssl:sslsocket() | gen_tcp:socket(), socket_type := tcp | ssl}.
+-type search_option() ::
+    {base, string()} |
+    {filter, filter()} |
+    {scope, scope()} |
+    {attributes, [string()]} |
+    {deref, dereference()} |
+    {types_only, boolean()} |
+    {timeout, integer()}.
+
+-doc "handle() - An opaque handle unique for the connection".
+-type handle() :: term().
+
+-doc "attribute() - `{Type = string(), Values=[string()]}`".
+-type attribute() :: {Type :: string(), Value :: [string()]}.
+
+-doc "modify_op() - See `mod_add/2`, `mod_delete/2`, `mod_replace/2`".
```

Comment:
Remove the initial "modify_op() - " in the doc. It should be just:
-doc "See `mod_add/2`, `mod_delete/2`, `mod_replace/2`".

Reply:
yes. let me fix it.